[Reviewed] [Autotyping] Simplify the description

D8H opened the pull request:
* Don't show in changelog

tristanbob commented:
Did you create a wiki page to share the info you are deleting?  (I didn't see the link to it)

D8H commented:
> Did you create a wiki page to share the info you are deleting? (I didn't see the link to it)

I moved useful information in other descriptions that better fit the purpose and removed what is obvious.

For instance:
- I removed "Change the typing speed with the Change interval action" because this is action description is already "Change the time between characters being typed.
- I moved "When the text changes, the automatic typing starts again from the beginning with the new text" in the "Start at the beginning of the text" action description because this is where users need to know that calling this action maybe not be necessary. If they read the extension description entirely (which I doubt), I bet they wouldn't remember it at this point.

D8H commented on extensions/reviewed/AutoTyping.json:
```diff
@@ -7,22 +7,12 @@
-    "Animate text to simulate it being written one character at at time (a \"typewriter\" effect), with a customizable time between each character. Useful for dialogue scenes or visual novels.",
+    "Reveal a text one letter after the other (a \"typewriter\" effect).",
     "",
-    "How to use:",
-    "- Add the behavior to a Text object (BBText, Bitmap Text object) and choose the interval between characters.",
```
It's unlikely someone try to use the behavior on a sprite.
It can be nice to explain it in a documentation but I think it's not mandatory.

D8H commented on extensions/reviewed/AutoTyping.json:
```diff
@@ -7,22 +7,12 @@
-    "- Add the behavior to a Text object (BBText, Bitmap Text object) and choose the interval between characters.",
-    "",
-    "Tips:",
-    "- Change the typing speed with the \"Change interval\" action.",
-    "- Pause the typing and resume it at any time.",
-    "- Use the \"Skip to end of text\" action to give users a way to skip text they don't want to read ",
```
This is a listing of the actions. Their descriptions are already clear.

D8H commented on extensions/reviewed/AutoTyping.json:
```diff
@@ -663,7 +653,7 @@
         },
         {
-          "description": "Start at the beginning of text.",
+          "description": "Start at the beginning of text. When the text changes, the automatic typing starts again from the beginning with the new text.",
```
I moved this explanation on the action directly.

D8H commented on extensions/reviewed/AutoTyping.json:
```diff
@@ -7,22 +7,12 @@
-    "- Pause the typing and resume it at any time.",
-    "- Use the \"Skip to end of text\" action to give users a way to skip text they don't want to read ",
-    "- If the text changes, the automatic typing starts again from the beginning with the new text.",
-    "- Use a condition to check if the typing finished.",
-    "- Use a condition to check if a new text character was added. This is useful to trigger audio effects.",
-    "- Use the \"CurrentCharacter\" expression to know when a specific character was displayed.",
```
This is a listing of the condition and expression. Their descriptions are already clear.

D8H reviewed (commented):
@tristanbob I added comments to explain the changes. Is there any information missing?

D8H commented:
Should we go ahead with this?

tristanbob reviewed (approved):
Yes I approve.
